war/src/main/scss/base/spacing.scss

```diff
@@ -0,0 +1,49 @@
+$unit: 0.4rem;
+
+@for $i from 1 through 6 {
+  .jenkins-\!-margin-#{value} {
+    margin: $i * $unit !important;
+  }
```

Comment:
Something seems to be going wrong here. The generated CSS file in your version has `.jenkins-\!-margin-top-value` rather than e.g. `.jenkins-\!-margin-left-2` as expected.

Reply:
Fixed :)